packages/salesforcedx-vscode-lwc-next/package.nls.json

```diff
@@ -1,3 +1,3 @@
 {
-  "force_lightning_lwc_create_text": "SFDX: Create LWC Bundle"
+  "force_lightning_lwc_create_text": "SFDX: Create Lightning Web Component"
```

Comment:
Change the text to match what will be on the command line on the SFDX CLI.
This new command will be available in Summer '18 so for now, this is done in the vscode-lwc-next extension.
When Summer rolls out, we will push this out.